indico/modules/rb/blueprint.py

```diff
@@ -0,0 +1,161 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from indico.modules.rb.controllers.admin import reservations as reservation_admin_handlers
+from indico.modules.rb.controllers.user import (
+    blockings as blocking_handlers,
+    index as index_handler,
+    photos as photo_handlers,
+    reservations as reservation_handlers,
+    rooms as room_handlers
+)
+from indico.web.flask.wrappers import IndicoBlueprint
+
+
+rooms_blueprint = _bp = IndicoBlueprint('rooms', __name__, url_prefix='/rooms', template_folder='templates')
+
+
+# Photos
+_bp.add_url_rule('/room/<roomLocation>/<int:roomID>/photo-<any(small,large):size>.jpg', 'photo',
+                 photo_handlers.room_photo)
+
+
+# Home, map, lists, search
+_bp.add_url_rule('/',
```

Comment:
I think that way of splitting the url registrations doesn't really improve readability